addons/account/wizard/account_merge_wizard.py

```diff
@@ -0,0 +1,319 @@
+from odoo import _, api, fields, models, Command
+from odoo.exceptions import UserError
+from odoo.tools import SQL
+
+
+class AccountMergeWizard(models.TransientModel):
+    _name = 'account.merge.wizard'
+    _description = "Account merge wizard"
+
+    account_ids = fields.Many2many('account.account')
+    is_group_by_name = fields.Boolean(
+        string="Group by name?",
+        default=True,
+        help="Tick this checkbox if you want accounts to be grouped by name for merging."
+    )
+    wizard_line_ids = fields.One2many(
+        comodel_name='account.merge.wizard.line',
+        inverse_name='wizard_id',
+        compute='_compute_wizard_line_ids',
+        store=True,
+        readonly=False,
+    )
+    disable_merge_button = fields.Boolean(compute='_compute_disable_merge_button')
+
+    @api.model
+    def default_get(self, fields):
+        res = super().default_get(fields)
+        if not set(fields) & {'account_ids', 'wizard_line_ids'}:
```

Comment:
Because since `account_ids` isn't in the view, it doesn't appear in `fields`. Of course, I could add it to the view if that's better.
![image](https://github.com/user-attachments/assets/49abd647-ee94-4d04-8004-4592ece41f1c)